data/systemd/yggdrasil-sysuser.conf.in

```diff
@@ -0,0 +1 @@
+u @user@ - "yggdrasil system user" @sharedstatedir@/yggdrasil -
```

Comment:
Fair point about the header comment. I didn't feel it was necessary for a single line file, but it does make the file format more explicit.

According to [the documentation](https://www.freedesktop.org/software/systemd/man/latest/sysusers.d.html#Shell):

> The login shell of the user. If not specified, this will be set to /usr/sbin/nologin, except if the UID of the user is 0, in which case /bin/sh will be used.
> 
> Only applies to lines of type u and should otherwise be left unset (or "-"). It is recommended to omit this, unless a shell different /usr/sbin/nologin must be used.

I omitted it from the file specifically so that the default value is used.